README.md

```diff
@@ -27,6 +28,15 @@ Page](https://mana.mozilla.org/wiki/display/EN/careers.mozilla.org)
 docker-compose build
 ```
 
+### Create local `.env` file
+
+You _probably_ want to add a couple local environment settings to improve local dev:
+
+1. In the root of the project, create a file named `.env`
+2. Add `DEBUG=True` - this will prevent robots from indexing the site if you happen to build locally and push to a URL that shouldn't be indexed.
```

Comment:
Not sure what you mean here. If you're pushing to a public instance you most probably don't want DEBUG=True, even if it's dev. You can use `ENGAGE_ROBOTS=False` if you don't want the instance to get indexed.

Reply:
to show in-progress work, i've built the site locally and pushed to a subdomain on my personal hosting. originally had this specifying `ENGAGE_ROBOTS`, but tried to get fancy. will revert.

Reply:
Thanks!